index.html

```diff
@@ -7,6 +7,30 @@
   <title>Hello Front-End</title>
 </head>
 <body>
-  <h1>Hello Front-End</h1>
+
+  <main>
+    <h1>Meet Guidebooks</h1>
+    <h2>Discover hundered of spots recommended by Airbnb hosts</h2>
+    <div>
+      <img src="img/san-francisco.jpg">
+      <img src="img/new-york.jpg">
+      <img src="img/london.jpg">
```

Comment:
An alternate approach to using an `<img>` tag here would be to set that image as the `background-image` of a `<div>`. Then the only HTML content of the `<div>` would be the text (e.g., "London", "New York"), which you can center horizontally and vertically using flexbox.

So something like...

```html
<div>
  <span>San Francisco</span>
</div>
```
```css
div {
  display: flex;
  justify-content: center;
  align-items: center;
  background-image: url("whatever the url of the image is");
}
```